l10n/css.json

```
@@ -1575,7 +1616,8 @@
     "en-US": "a simple list of ",
     "fr": "une simple liste de ",
     "ja": "単純なリスト: ",
-    "ru": "простой список из "
+    "ru": "простой список из ",
+    "zh-CN": "简单的列表："
```

context: https://developer.mozilla.org/zh-CN/docs/Web/CSS/transform-origin#%E8%A7%84%E8%8C%83

```suggestion
    "zh-CN": "由以下类型组成的简单列表："
```